src/kudu/consensus/consensus_meta.cc

```diff
@@ -200,33 +220,40 @@ const string& ConsensusMetadata::leader_uuid() const {
 
 LastKnownLeaderPB ConsensusMetadata::last_known_leader() const {
   DFAKE_SCOPED_RECURSIVE_LOCK(fake_lock_);
-  return last_known_leader_;
+  return pb_.last_known_leader();
 }
 
 std::map<int64_t, PreviousVotePB>
 ConsensusMetadata::previous_vote_history() const {
   DFAKE_SCOPED_RECURSIVE_LOCK(fake_lock_);
-  return previous_vote_history_;
+  std::map<int64_t, PreviousVotePB> pvh;
```

Comment:
since  you are returning the entire map, can't you just return pb_.previous_vote_history()

Reply:
I tried to but I need to convert from google::protobuf::Map somehow.